.ci/docker-compose-file/kafka/generate-certs.sh

```diff
@@ -10,10 +10,10 @@ HOST="*."
 DAYS=3650
 PASS="password"
 
-cd /var/lib/secret/
+cd /var/lib/secret
 
 # Delete old files
-(rm ca.key ca.crt server.key server.csr server.crt client.key client.csr client.crt server.p12 kafka.keystore.jks kafka.truststore.jks 2>/dev/null || true)
+rm -f ca.key ca.crt server.key server.csr server.crt client.key client.csr client.crt server.p12 kafka.keystore.jks kafka.truststore.jks
```

Comment:
this is to make sure the files are actually deleted, previous version would exit with 0 even if `rm` failed